src/Categories/Category/Monoidal/Star-Autonomous.agda

```diff
@@ -0,0 +1,35 @@
+{-# OPTIONS --without-K --safe #-}
+
+open import Categories.Category
+open import Categories.Category.Monoidal.Core using (Monoidal)
+open import Categories.Category.Monoidal.Symmetric
+
+module Categories.Category.Monoidal.Star-Autonomous {o ℓ e} {C : Category o ℓ e} (M : Monoidal C) where
+
+open import Level
+
+open import Categories.Category.Product
+open import Categories.Functor
+open import Categories.Functor.Properties
+open import Categories.NaturalTransformation.NaturalIsomorphism
+open import Categories.Morphism C
+open import Categories.Functor.Hom
+open Hom C
+
+open Category C renaming (op to Cᵒᵖ) hiding (id)
+open Monoidal M
+open Functor ⊗ renaming (op to ⊗ₒₚ)
+
+open import Relation.Binary.Structures
+
+record Star-Autonomous : Set (levelOfTerm M) where
+  field
+    symmetric : Symmetric M
+    F⁻¹ : Functor Cᵒᵖ C
```

Comment:
I would call `F⁻¹` something like `star` or `dual` (with capitalisation dependent on agda-categories conventions for functors).

Reply:
That makes more sense! Thanks!